entrypoint-monitoring/lambda/src/lib/EnvConfigBuilder.ts

```diff
@@ -0,0 +1,27 @@
+import Config from "../types/Config";
+import {LogLevel} from "./ScanLogger";
+
+export default class EnvConfigBuilder {
+
+  protected getEnvVar(name: string): string {
+    if (process.env[name] === undefined) {
+      throw new Error(`The mandatory environment variable is missing: ${name}`)
+    }
+
+    return process.env[name]
+  }
+
+  protected getOptionalEnvVar(name: string): string|undefined {
+    return process.env[name]
+  }
+
+  build(): Config {
+    return {
+      region: this.getEnvVar('AWS_REGION'),
```

Comment:
@DmitriiTartynovLCM Unsure if it's just me, and don't think this will be a problem in AWS itself, but I was getting errors when running this Lambda locally that the region was missing. I was able to get it working by setting the region in config.ts.

Reply:
@tom-fleming That's right. The region must be set in the config when executed on local. This environment variable is set by AWS for all lambdas according to docs.